src/Barberry/ContentType.php

```diff
@@ -59,7 +59,7 @@ class ContentType
         'ots' => 'application/vnd.oasis.opendocument.spreadsheet-template',
         'ods' => 'application/vnd.oasis.opendocument.spreadsheet',
         'txt' => 'text/plain',
-        'xls' => 'application/vnd.ms-excel',
+        'xls' => array ('application/vnd.ms-excel', 'application/vnd.ms-office', 'application/CDFV2'),
```

Comment:
Spacing? And maybe convert to short syntax everywhere?)

Reply:
Improved. Ready for review.